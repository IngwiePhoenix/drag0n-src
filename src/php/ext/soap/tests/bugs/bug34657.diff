002+ SOAP-ERROR: Parsing WSDL: Couldn't find <definitions> in 'http://i_dont_exist.com/some.wsdl'
002- SOAP-ERROR: Parsing WSDL: Couldn't load from 'http://i_dont_exist.com/some.wsdl'%A