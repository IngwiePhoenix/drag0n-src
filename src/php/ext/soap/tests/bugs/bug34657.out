SoapFault
SOAP-ERROR: Parsing WSDL: Couldn't find <definitions> in 'http://i_dont_exist.com/some.wsdl'
ok